### PR TITLE
Python 3 support & covering with unittests

### PR DESCRIPTION
**Pull request by vshlapakov**
Review, please.

---
This change is [Reviewable](https://reviewable.io/reviews/scrapinghub/python-scrapinghub/26)

### CODE REVIEW

File: tests/test_scrapinghub.py [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+
+def test_job_add_params(job):
+    assert job._add_params({'param': 'value'}) == {
+        'param': 'value', 'job': '1/2/3'}
+
+# --------------------- Other tests ---------------------
+
+
+def test_apierror():
+    assert issubclass(APIError, Exception)
```
Comment (chekunkov) [inline tests/test_scrapinghub.py]: what's the point?

File: tests/test_scrapinghub.py [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+def test_jobset_add_params_void(jobset):
+    assert jobset._add_params({}) == {'param': 'value'}
+
+
+def test_jobset_add_params_base(jobset):
+    params = {'paramX': 'valueX', 'param': 'newvalue'}
+    assert jobset._add_params(params) == {
+            'param': 'newvalue', 'paramX': 'valueX'}
+
+# --------------------- Job test section ---------------------
```
Comment (chekunkov) [inline tests/test_scrapinghub.py]: let's create tests package instead

```
tests/
    __init__.py
    conftest.py
    test_job.py
    test_connection.py
```

**Comment by vshlapakov** [comment]
Splitted a test module to a test package, cleaned everything and squashed, ready for final review.

**Comment by chekunkov** [comment]
LGTM

**Pull request merged**